components/patient-data/impl/src/test/java/org/phenotips/data/internal/controller/VariantListControllerTest.java

```diff
@@ -470,7 +470,7 @@ public void writeJSONAddsContainerWithOnlySelectedFields() throws ComponentLooku
 
         Map<String, String> item = new LinkedHashMap<>();
         item.put(VARIANT_KEY, "variantName");
-        item.put(GENESYMBOL_KEY, "geneSymbol");
+        item.put(GENE_KEY, "geneSymbol");
```

Comment:
Shouldn't the value be updated as well? I see two other places in this file with the same problem.